Makefile

```diff
@@ -70,13 +70,18 @@ else
 endif
 
 test-with-kind: fixture intel-sgx-admissionwebhook intel-fpga-admissionwebhook intel-deviceplugin-operator install-tools
-	# Build a Cluster with KinD & Install Cert-Manager 
+	# Build a Cluster with KinD & Load Images & Install Cert-Manager
 	kind create cluster
-	kind load docker-image intel/intel-sgx-admissionwebhook:devel intel/intel-fpga-admissionwebhook:devel intel/intel-deviceplugin-operator:devel
+	docker tag $(REG)intel-sgx-admissionwebhook:$(TAG) intel/intel-sgx-admissionwebhook:devel
+	docker tag $(REG)intel-fpga-admissionwebhook:$(TAG) intel/intel-fpga-admissionwebhook:devel
+	docker tag $(REG)intel-deviceplugin-operator:$(TAG) intel/intel-deviceplugin-operator:devel
```

Comment:
can you check if `REG` and `TAG` can be unset for the particular Jenkins stage that runs `make test-with-kind`? this extra tagging is specific to the Jenkins case only....

Reply:
Yeah,, I see. It's the reason why I said it looks "dirty".
Well, let me try to find a way to change. But,, it takes too long time to see the result from Jenkinsfile.
Olev is on vacation now...